addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4525,6 +4525,12 @@ public OnClientDisconnect(client)
 	FF2flags[client]=0;
 	Damage[client]=0;
 	uberTarget[client]=-1;
+	
+	if(MusicTimer[client]!=INVALID_HANDLE)
```

Comment:
@50Wliu 

MusicTimer[client] will be invalid when after Timer Callback. But not INVALID_HANDLE.

Example: `MusicTimer[client]=CreateTimer(time, PlayBGM);`
Then, after time? 

It will be `MusicTimer[client]=CreateTimer(time, Timer_PrepareBGM, GetClientUserId(client), TIMER_FLAG_NO_MAPCHANGE);` by Timer_PrepareBGM or Nothing.

Nothing? Yeah. This will be KillTimer Error. So, KillTimer before timer end.